flow/designs/asap7/jpeg_lvt/config.mk

```diff
@@ -0,0 +1,18 @@
+export PLATFORM               = asap7
```

Comment:
What is the goal for this new design?  Should it be in the regression?

Reply:
@maliberty Sorry I forgot to add one line to this file when cleaning up various PRs and consolidating them into this one. This test is designed to test/prove that LVT cells are used by the resizer when available. @tspyrou and I agreed that this would be a good first article test for this functionality (min implant rules aside).

Reply:
Never mind. I already had that line in the file.

export TNS_END_PERCENT       ?= 100

Reply:
Did you 'test/prove' that it uses a Vt mix?  Afaik this switches to only using LVT cells so there is nothing else for the resizer to choose from and no mix.

Reply:
I did not know that. I will double check and add EXTRA_LIBS should that prove to be the case.  I did check that the results were different and better.

Reply:
I think you might want to try adding it to `ADDITIONAL_LIBS`

Reply:
Still learning the various magic variables. Thanks @rovinski

Reply:
Different and better makes sense since you made everything low Vt.  You need to verify a mix of Vt

Reply:
Works fine as expected

    **1    2.08                           _038059_ (net)
                 19.72    0.03  922.13 ^ _101202_/A (FAx1_ASAP7_75t_R)
                 49.76   29.77  951.90 v _101202_/CON (FAx1_ASAP7_75t_R)
     2    3.62                           _038090_ (net)
                 30.83   20.31  972.21 ^ _101202_/SN (FAx1_ASAP7_75t_R)
     1    0.68                           _012813_ (net)**
                 _30.83    0.00  972.21 ^ _095335_/A (INVx1_ASAP7_75t_L)
                 14.55   10.53  982.74 v _095335_/Y (INVx1_ASAP7_75t_L)_
     **1    1.20                           _038061_ (net)
                 14.55    0.03  982.78 v _109085_/A (HAxp5_ASAP7_75t_R)
                 71.35   59.82 1042.59 v _109085_/SN (HAxp5_ASAP7_75t_R)**
     _3    3.48                           _003000_ (net)
                 71.35    0.02 1042.62 v _074622_/A (INVx1_ASAP7_75t_L)
                 23.06   17.01 1059.63 ^ _074622_/Y (INVx1_ASAP7_75t_L)_